rebac-admin-backend/v1/response_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2024 Canonical Ltd.
+
+package v1
+
+import (
+	"errors"
+	"net/http"
+	"testing"
+
+	"github.com/canonical/identity-platform-admin-ui/rebac-admin-backend/v1/resources"
+	qt "github.com/frankban/quicktest"
```

Comment:
i understand that this will be a repo on its own eventually, but for now can we stick with `uber/gomock` and std library for testing?

Reply:
-1 on using the std library.. quicktest serves use very well in all of our projects AND we know the author :)

Reply:
but imports still need sorting